src/main/java/com/ebivariation/contigalias/dus/FTPBrowser.java

```diff
@@ -10,22 +10,24 @@
 
 public class FTPBrowser {
 
-    public static final int FTP_PORT = 21;
+    private final Logger logger = LoggerFactory.getLogger(FTPBrowser.class);
 
-    public static final String NCBI_FTP_SERVER = "ftp.ncbi.nlm.nih.gov";
+    private final FTPClient ftp = new FTPClient();
 
-    private final Logger logger = LoggerFactory.getLogger(FTPBrowser.class);
+    public void connect(String address, Integer port) throws IOException {
 
-    private FTPClient ftp = new FTPClient();
+        // If port is not specified (null) then set it to 21.
+        if (port == null) {
```

Comment:
rather than accepting a null port, create another method that calls this one with the default port:
```
    public void connect(String address) {
        connect(address, 21);
    }
```
and either make the port an int or throw an exception if it's null.

Reply:
also, the constant with the port should probably be in this class, not in NCBIBrowser.